src/SFA.DAS.AdminService.Web/Services/Gateway/PeopleInControlOrchestrator.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using SFA.DAS.AdminService.Web.Infrastructure.Apply;
+using SFA.DAS.AdminService.Web.Infrastructure.RoatpClients;
+using SFA.DAS.AdminService.Web.Models;
+using SFA.DAS.AdminService.Web.ViewModels.Roatp.Gateway;
+using SFA.DAS.AssessorService.ApplyTypes.Roatp;
+
+namespace SFA.DAS.AdminService.Web.Services.Gateway
+{
+    public class PeopleInControlOrchestrator: IPeopleInControlOrchestrator
+    {
+        private readonly IRoatpApplicationApiClient _applyApiClient;
+        private readonly IRoatpOrganisationSummaryApiClient _organisationSummaryApiClient;
+        private readonly ILogger<PeopleInControlOrchestrator> _logger;
+
+        public PeopleInControlOrchestrator(IRoatpApplicationApiClient applyApiClient, IRoatpOrganisationSummaryApiClient organisationSummaryApiClient, ILogger<PeopleInControlOrchestrator> logger)
+        {
+            _applyApiClient = applyApiClient;
+            _logger = logger;
+            _organisationSummaryApiClient = organisationSummaryApiClient;
+        }
+
+        public async Task<PeopleInControlPageViewModel> GetPeopleInControlViewModel(GetPeopleInControlRequest request)
+        {
+             _logger.LogInformation($"Retrieving people in control details for application {request.ApplicationId}");
+
+             var model = new PeopleInControlPageViewModel();
+            await model.PopulatePageCommonDetails(_applyApiClient, request.ApplicationId, GatewayPageIds.PeopleInControl, request.UserName,
+                RoatpGatewayConstants.Captions.PeopleInControlChecks,
+                RoatpGatewayConstants.Headings.PeopleInControl,
+                NoSelectionErrorMessages.PeopleInControlCheck);
+
+            _logger.LogInformation($"Retrieving people in control - type of organisation for application {request.ApplicationId}");
+            model.TypeOfOrganisation = await _organisationSummaryApiClient.GetTypeOfOrganisation(request.ApplicationId);
+
+
+            _logger.LogInformation($"Retrieving people in control - company directors for application {request.ApplicationId}");
+            model.CompanyDirectorsData = new PeopleInControlData
```

Comment:
Could the Captions and External Source Headings be put into a constant string or lookup somewhere?